glossary.md

```diff
@@ -0,0 +1,89 @@
+# Glossary of Optimistic Ethereum Terms
+
+### Layer 1 (L1)
+
+The Ethereum Mainnet.
+
+### Layer 2 (L2)
+
+The Optimistic Ethereum Network.
+
+### State Root Oracle
```

Comment:
Is this supposed to be the block hash oracle? Also could link to this? https://eips.ethereum.org/EIPS/eip-210

Reply:
Yes, it's the same as the `Block Hash Oracle`, but we've changed the name to this (actually I think we're even shifting towards `L2 State Root Oracle`. 
Yes, it provides block hashes, but the purpose for it is more closely tied to getting state roots for L2.

Reply:
```suggestion
### L2 State Root Oracle
```